packages/web/src/features/users/ResetPasswordView.tsx

```diff
@@ -0,0 +1,105 @@
+import ModalView, { ModalViewTitle } from '../../shared/components/ModalView';
+import FormLabel from '../../shared/components/form/FormLabel';
+import InputError from '../../shared/components/form/InputError';
+import SubmitButton from '../../shared/components/form/SubmitButton';
+import TextInput from '../../shared/components/form/TextInput';
+import { useForm } from 'react-hook-form';
+import { useFlash } from '../../shared/hooks/flash';
+import apiClient from '../../shared/apiClient';
+import { ApiClientError } from '@translation/api-client';
+import Form from '../../shared/components/form/Form';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import useAuth from '../../shared/hooks/useAuth';
+import useTitle from '../../shared/hooks/useTitle';
+
+export default function ResetPasswordView() {
+  const { t } = useTranslation(['users', 'common']);
+  useTitle(t('users:reset_password'));
+
+  const { refreshAuth } = useAuth();
+
+  const [search] = useSearchParams();
+  const token = search.get('token') ?? undefined;
```

Comment:
It would be good to have a useEffect that confirms there is a token, and if not shows an error message and redirects to the home page.

Reply:
Done. In development mode, this flashes twice due to the strict mode setting. I don't think this is a problem since we immediately navigate away from `/reset-password` on the error.